howtos/manage_cpp_standard.rst

```diff
@@ -6,27 +6,25 @@ How to manage C++ standard [EXPERIMENTAL]
 .. warning::
 
     This is an **experimental** feature subject to breaking changes in future releases.
+    Previously, it was implemented as a first level setting ``cppstd``, we encourage
+    you to adopt the new subsetting and update your recipes if they were including the
+    deprecated one in its :ref:`settings_property` attribute.
 
-The setting representing the C++ standard is ``cppstd``.
-The detected default profile doesn't set any value for the ``cppstd`` setting.
+
+The setting representing the C++ standard is ``compiler.cppstd``.
+The detected default profile doesn't set any value for the ``compiler.cppstd`` setting, but
+Conan will use the default for each compiler if no value is provided.
```

Comment:
Not accurate. Conan won't do anything if no value is specified. But the compiler will use, of course, the default value.

Reply:
That's right, and very important!